src/lib/cli.rs

```diff
@@ -111,11 +115,11 @@ fn run(cli_args: CliArgs, global_config: &GlobalConfig) {
 
     let crate_name = envmnt::get_or("CARGO_MAKE_CRATE_NAME", "");
     if crate_name.len() > 0 {
-        info!("Project: {}", &crate_name);
```

Comment:
let keep in info.
when working in workspace, it really helps to see this for each member as it tells you that you finished previous one and started this one.